core/docs/spark-profiling-tool.md

```diff
@@ -626,6 +626,8 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
   -p, --print-plans               Print the SQL plans to a file named
                                   'planDescriptions.log'.
                                   Default is false.
+      --platform  <arg>           Cluster platform where Spark CPU workloads were
+                                  executed. Default is onprem.
```

Comment:
the markdown needs to be updated to reflect the actual help message from the scala code.
```
Options include onprem, dataproc, emr, databricks.
```

Reply:
Updated